pkg/resources/common/formatter.go

```diff
@@ -212,6 +220,47 @@ func excludeFields(request *types.APIRequest, unstr *unstructured.Unstructured)
 	}
 }
 
+func convertMetadataFields(request *types.APIRequest, gvk schema2.GroupVersionKind, unstr *unstructured.Unstructured) {
+	if request.Schema != nil {
+		cols := GetColumnDefinitions(request.Schema)
+		for _, col := range cols {
+			gvkDateFields, gvkFound := DateFieldsByGVKBuiltins[gvk]
+			if col.Type == "date" || (gvkFound && slices.Contains(gvkDateFields, col.Name)) {
+				index := GetIndexValueFromString(col.Field)
+				if index == -1 {
+					logrus.Errorf("field index not found at column.Field struct variable: %s", col.Field)
+					return
+				}
+
+				curValue, got, err := unstructured.NestedSlice(unstr.Object, "metadata", "fields")
+				if err != nil {
+					logrus.Errorf("failed to get metadata.fields slice from unstr.Object: %s", err.Error())
+				}
+
+				if !got {
+					logrus.Debugf("couldn't find metadata.fields at unstr.Object")
+					return
+				}
+
+				millis, err := strconv.ParseInt(curValue[index].(string), 10, 64)
```

Comment:
I think your test data isn't old enough, because I'm running into errors with some apps I created this morning:
```
ERRO[0020] failed to convert timestamp value: strconv.ParseInt: parsing "6h7m": invalid syntax 
```

And when I read the code, why is this code not converting strings like `6h` into a numeric time and why are you writing the values back as human durations and not absolute timestamps?

It looks like the converter you wrote in virtual/virtual.go isn't kicking in.